documentation/trunk/solutions/basic-computing/eiffelbase/eiffelbase-tutorial/eiffelbase-kernel.wiki

```diff
@@ -112,13 +111,16 @@ A number of classes offer facilities which are very close to the language level.
 
 ==Basic types==
 
-The basic types [[ref:libraries/base/reference/boolean_chart|BOOLEAN]] , [[ref:libraries/base/reference/character_8_chart|CHARACTER]] , [[ref:libraries/base/reference/integer_32_chart|INTEGER]] , [[ref:libraries/base/reference/real_32_chart|REAL]]  and [[ref:libraries/base/reference/real_64_chart|DOUBLE]]  are defined by classes of the Kernel library. <br/>
-In reading the class specifications for the numeric types [[ref:libraries/base/reference/integer_32_chart|INTEGER]] , [[ref:libraries/base/reference/real_32_chart|REAL]]  and [[ref:libraries/base/reference/real_64_chart|DOUBLE]] , you might think that the type declarations are too restrictive. For example the addition operation in class [[ref:libraries/base/reference/real_32_chart|REAL]]  reads 
+The basic types [[ref:libraries/base/reference/boolean_chart|BOOLEAN]] ,
+[[ref:libraries/base/reference/character_8_chart|CHARACTER]] ,
+[[ref:libraries/base/reference/integer_32_chart|INTEGER]] , and [[ref:libraries/base/reference/real_32_chart|REAL]] are defined by classes of the Kernel library. <br/>
+In reading the class specifications for the numeric types
```

Comment:
I noticed you removed the REAL_64 case.
Why not, but we could add mention to the INTEGER_(8 -> 64).. REAL_(32,64).. classes, and the existing alias/map  for INTEGER -> INTEGER_32, REAL -> REAL_32, DOUBLE -> REAL_64.